lib/util.py

```diff
@@ -122,3 +123,29 @@ def shell_command(command):
     else:
         return out, err
 
+# ## Many of the view functions just do a print without any valid returns
```

Comment:
I think all view functions behave this way, it is a CLI application the views are supposed to do the rendering.